### PR TITLE
Added localizations for undefined audio track language and no linguistic content

### PR DESCRIPTION
**Pull request by MarkEPAM**
Added localizations for undefined audio track language and no linguistic content.
Created constant object in AudioAndCCSelectionPanel.js for strings.

### CODE REVIEW

**Review by brol1dev — APPROVED** [review]
LGTM

File: sdk/react/panels/AudioAndCCSelectionPanel.js [hunk 1]
```diff
@@ -56,13 +64,20 @@ var AudioAndCCSelectionPanel = React.createClass({
   },
 
   onAudioTrackSelected: function(name) {
-    if (this.props.selectedAudioTrackTitle !== name) {
-      this.props.onSelectAudioTrack(name);
+    let localizedTitleForUndefinedLanguage = Utils.localizedString(this.props.config.locale, stringConstants.undefinedLanguageTitle, this.props.config.localizableStrings);
+    let localizedTitleForNoLinguisticContent = Utils.localizedString(this.props.config.locale, stringConstants.noLinguisticContentTitle, this.props.config.localizableStrings);
+    var originalName = name;
```
Comment (pilievOoyala) [inline sdk/react/panels/AudioAndCCSelectionPanel.js:69]: I think you have the naming of these two variable backwards. If we edit the name, it's no longer the original name. I would rename the param that's passed in to be 'originalName', and the new localized one to be 'name'.

Comment (pilievOoyala) [reply sdk/react/panels/AudioAndCCSelectionPanel.js:69]: Disregard the previous statement. I got the parameters mixed up.

File: sdk/react/panels/AudioAndCCSelectionPanel.js [hunk 2]
```diff
@@ -56,13 +64,20 @@ var AudioAndCCSelectionPanel = React.createClass({
...
+
+    originalName = originalName.replace(localizedTitleForUndefinedLanguage, stringConstants.undefinedLanguageTitle);
+    originalName = originalName.replace(localizedTitleForNoLinguisticContent, stringConstants.noLinguisticContentTitle);
+
+    if (this.props.selectedAudioTrackTitle !== originalName) {
+      this.props.onSelectAudioTrack(originalName);
```
Comment (pilievOoyala) [inline sdk/react/panels/AudioAndCCSelectionPanel.js:75]: I cannot tell what the different is between onSelectAudioTrack and onAudioTrackSelected.  I can see what this function is doing but if I were to read these two function names someone else in the code, I would not know which one to use. Can you explain the difference and maybe we can name them better?

**Comment by pilievOoyala** [comment]
LGTM

File: sdk/react/panels/AudioAndCCSelectionPanel.js (hunk 2 above)
Comment (pilievOoyala) [reply sdk/react/panels/AudioAndCCSelectionPanel.js:75]: We can do this in another PR.

**Pull request merged**